app/experimenter/experiments/tests/api/v4/test_serializers.py

```diff
@@ -88,3 +92,87 @@ def test_serializer_outputs_expected_schema(self):
                 {"ratio": 33, "slug": "control", "value": None},
             ],
         )
+
+    def test_serializer_outputs_expected_schema_with_nameSpace_bucket(self):
+        audience = "us_only"
+        features = ["pinned_tabs", "picture_in_picture"]
+        normandy_slug = "experimenter-normandy-slug"
+        today = datetime.datetime.today()
+        experiment = ExperimentFactory.create(
+            type=Experiment.TYPE_RAPID,
+            rapid_type=Experiment.RAPID_AA_CFR,
+            audience=audience,
+            features=features,
+            normandy_slug=normandy_slug,
+            firefox_min_version="80.0",
+            proposed_enrollment=9,
+            proposed_start_date=today,
+        )
+
+        ExperimentVariantFactory.create(
+            experiment=experiment, slug="control", is_control=True
+        )
+        ExperimentVariantFactory.create(experiment=experiment, slug="variant-2")
+
+        ExperimentBucketNamespace.request_namespace_buckets(
+            experiment.normandy_slug,
+            experiment,
+            NIMBUS_DATA["ExperimentDesignPresets"]["empty_aa"]["preset"]["arguments"][
+                "bucketConfig"
+            ]["count"],
```

Comment:
Actually this doesn't even need to call this here, all teh inputs to this test can just be hard coded, the place where we'd want to test this is on the task, below.